.github/workflows/csslint.yaml

```
@@ -12,4 +12,10 @@ jobs:
         run: npm install
       - name: Lint
         # Run StyleLint on diffed CSS files. Do not run if there are no diffs.
-        run: git diff --diff-filter=ACM --name-only origin/master HEAD -- . ':!node_modules' | grep -E "(.*)\.(css|htm|html)$" | xargs --no-run-if-empty npx stylelint
+        run: |
+          git diff --diff-filter=ACM --name-only origin/master HEAD -- . ':!node_modules' | grep -E "(.*)\.(css|htm|html)$" | xargs --no-run-if-empty npx stylelint --fix
+          git config user.name "CSS Presubmit"
+          git add .
+          if [ git commit -m "Format CSS files" ]; then
+            git push origin `git rev-parse --abbrev-ref HEAD`
```

Will this work? I think it's a bit clearer. If so, can you change the other files, as well?

```suggestion
            git push origin `git branch --show-current`
```

---

It is clearer, but only works in version 2 of git, which might not be installed. At least it's not installed locally.

---

I'll try it and if it fails to work I'll revert it to the less clear version.